### PR TITLE
[BUG] Footer overlapping on widgets (Issue #18)

### PR DESCRIPTION
**Pull request by Zume-z**
Expected: Footer should not overlap. Issue https://github.com/yashchaudhari008/minime/issues/18.

Before:

https://github.com/yashchaudhari008/minime/assets/36926781/71fca3be-9360-41c0-a548-4ff0e87bb93a

After: 

https://github.com/yashchaudhari008/minime/assets/36926781/e2427552-b21a-48e8-9bad-f326295aefa5

### CODE REVIEW

File: src/app.module.scss [hunk 1]
```diff
@@ -1,8 +1,5 @@
 .app {
 	height: 100%;
-	gap: 16px;
-	padding: 32px;
-
```
Comment (yashchaudhari008) [inline src/app.module.scss:5]: Keep this as it is
and added `padding-bottom : 0` to remove padding from bottom

File: src/components/Footer/footer.module.scss [hunk 2]
```diff
@@ -6,10 +6,11 @@
 	display: flex;
 	justify-content: flex-end;
 	align-items: flex-end;
-	position: fixed;
+	position: relative;
```
Comment (yashchaudhari008) [inline src/components/Footer/footer.module.scss:9]: We can completely removed this property
```suggestion
```

File: src/components/Footer/footer.module.scss [hunk 3]
```diff
@@ -6,10 +6,11 @@
...
 	bottom: 0;
 	right: 0;
 	padding-right: 24px;
+
```
Comment (yashchaudhari008) [inline src/components/Footer/footer.module.scss:13]: extra line can be removed

File: src/components/TopBar/topBar.module.scss [hunk 4]
```diff
@@ -3,4 +3,5 @@
 .topBar {
 	@include font-poppins;
 	width: 100%;
+	padding: 32px 32px 16px;
```
Comment (yashchaudhari008) [inline src/components/TopBar/topBar.module.scss:6]: This is not need if gap and padding is maintained

**Review by yashchaudhari008 — CHANGES_REQUESTED** [review]
And finally, in `widgetHolderWrapper` you could add 
```
    flex: 1;
```
This will fix the issue ig

**Comment by Zume-z** [comment]
Thanks for feedback - appreciate it. Made the updates.

File: src/components/WidgetHolder/widgetHolder.module.scss [hunk 5]
```diff
@@ -2,7 +2,9 @@
 @import "/src/styles/variables.scss";
 
 .widgetHolderWrapper {
-	padding: 8px;
+	padding: 8px 40px;
+	height: 100%;
```
Comment (yashchaudhari008) [inline src/components/WidgetHolder/widgetHolder.module.scss:6]:
```suggestion
	padding: 8px;
```

File: src/components/Footer/footer.module.scss [hunk 6]
```diff
@@ -6,7 +6,6 @@
 	display: flex;
 	justify-content: flex-end;
 	align-items: flex-end;
-	position: fixed;
 	bottom: 0;
 	right: 0;
```
Comment (yashchaudhari008) [inline src/components/Footer/footer.module.scss:10]: Also, as we have removed position: fixed, we could also remove this
```suggestion
```

**Review by yashchaudhari008 — APPROVED** [review]

**Comment by yashchaudhari008** [comment]
Thanks for contributing @Zume-z , make sure you look at other issues too. And don't forget to star the project

**Pull request merged**